packages/now-cli/src/util/dev/server.ts

```diff
@@ -1130,6 +1140,19 @@ export default class DevServer {
     }
   }
 
+  getExtensionlessFile = (path: string) => {
```

Comment:
Does the function really need to be bound to `this`?

```suggestion
  getExtensionlessFile(path: string): string | null {
```

Reply:
It save us passing several variables that are already bound to `this` (dev server). We already store `this.nowConfig` so I stored the stuff I needed there too.